numpy/lib/function_base.py

```diff
@@ -431,7 +431,7 @@ def _hist_bin_auto(x):
 
 def histogram(a, bins=10, range=None, normed=False, weights=None,
               density=None):
-    r"""
```

Comment:
Oops.  It messed up my syntax highlighting.  I honestly thought it was a mistake.